smartseq2_single_sample/tasks/hisat2.wdl

```diff
@@ -24,8 +26,49 @@ task HISAT2PE {
     cpu: "4"
```

Comment:
May as well give this the full 16g it has access to. Might make the sort faster if you let it have more memory.